temporal-sdk/src/main/java/io/temporal/internal/worker/ActivityTask.java

```diff
@@ -42,4 +45,8 @@ public PollActivityTaskQueueResponse getResponse() {
   public Functions.Proc getCompletionHandle() {
     return completionHandle;
   }
+
+  public String getIdentity() {
```

Comment:
This doesn't look like a correct place to pass this data. A worker is already aware of it's identity. It shouldn't be passed as a part of an object having information about a specific activity task.